layers/+completion/helm/README.org

```diff
@@ -180,6 +181,13 @@ In the =helm-bookmarks= buffer:
 | ~C-f~       | toggle filename location                     |
 | ~C-o~       | open the selected bookmark in another window |
 
+** Colors
+~SPC C l~ opens =helm-colors=.
```

Comment:
There is also a key binding <kbd>SPC h d F</kbd> for `helm-faces`, which might warrant mention here.

Reply:
Your right, thanks for the suggestion.

The same key binding with the `ivy` layer calls: `counsel-describe-face`
It doesn't show the faces with colors, but opening a face shows the same Emacs help buffer with the face properties. So I'll add that as well to the Ivy layer.

Maybe the heading could be called `Colors/Faces`.